oktapam/client/pam_sdk_client.go

```diff
@@ -0,0 +1,47 @@
+package client
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/atko-pam/pam-sdk-go/client/pam"
+)
+
+type SDKClientWrapper struct {
+	SDKClient *pam.APIClient
+	Team      string
+}
+
+func checkStatusCodeFromSDK(resp *http.Response, allowed ...int) (int, error) {
```

Comment:
I will add later in SDK to return httpStatusCode as part of the API error returned from sdk itself so that we don't need to check it from the raw http response.